src/CAREUI/misc/PaginatedList.tsx

```diff
@@ -129,20 +130,26 @@ interface ItemsProps<TItem> {
 const Items = <TItem extends object>(props: ItemsProps<TItem>) => {
   const { loading, items } = useContextualized<TItem>();
 
+  if (loading) {
+    return null;
+  }
+
   return (
-    <ul className={props.className}>
-      {loading && props.shimmer
-        ? Array.from({ length: props.shimmerCount ?? 8 }).map((_, i) => (
-            <li key={i} className="w-full">
-              {props.shimmer}
-            </li>
-          ))
-        : items.map((item, index, items) => (
-            <li key={index} className="w-full">
-              {props.children(item, items)}
-            </li>
-          ))}
-    </ul>
+    <Timeline className="rounded-lg bg-white p-2 shadow" name="log update">
+      <ul className={props.className}>
+        {loading && props.shimmer
+          ? Array.from({ length: props.shimmerCount ?? 8 }).map((_, i) => (
+              <li key={i} className="w-full">
+                {props.shimmer}
+              </li>
+            ))
+          : items.map((item, index, items) => (
+              <li key={index} className="w-full">
+                {props.children(item, items)}
+              </li>
+            ))}
+      </ul>
+    </Timeline>
```

Comment:
PagiantedList is a generic reusable component. Wrapping timeline here is not appropriate. Because not all Paginated List Items are Timelines, although all Timelines may be Paginated List Items.